### DISCUSSION TITLE
Toasts don't stack when using preact

### DISCUSSION THREAD

**Discussion by ar06x** [palantir/blueprint#6706] [Q&A] [answered]
In the documentation, when clicking a button to show a toast (eg: "Procure toast") multiple times in quick succession, the toast will stack up like so:
![image](https://github.com/palantir/blueprint/assets/45273698/74019e60-1b41-4452-bd1a-66219d7d5de7)

However, when trying to do this in my app with preact it only shows one at a time and one after the other:
![image](https://github.com/palantir/blueprint/assets/45273698/3aa69543-f38a-46fe-adbf-fefefb80d343)

There are not multiple toasts in one location, it's just *one* toast there. Once the timeout is up for that toast or the toast is closed, the next toast in the "queue" is shown. I do not like this behavior and would like for it to be like the documentation.

Is there a way to fix this? Or will I just have to switch to `react` instead?

**Comment by adidahiya** [comment]
@ar065 can you link to a code sandbox demonstrating the issue? I haven't used Preact so I'm not familiar with any common pitfalls that may be causing this problem.

**Reply by ar06x** [reply, marked as answer]
I've decided to move to react while leveraging millionjs. However [here](https://codesandbox.io/p/devbox/blueprintjs-preact-toast-issue-cqtxcl) is a quick and simple sandbox showing the issue. I believe it's due to preact's limitations and lack of features compared to react
